Example/Podfile

```diff
@@ -1,7 +1,15 @@
 source 'https://cdn.cocoapods.org/'
-platform :ios, '14.0'
+platform :ios, '16.0'
 use_frameworks!
 
+#post_install do |installer|
```

Comment:
Als het niet meer gebruikt wordt kan dit weg toch?

Reply:
Ik had hier misschien een comment boven moeten zetten, maar kan het ook weghalen. Met dit stukje code is het iets makkelijk om de deprecation warnings/errors in het example project naar boven te krijgen

Reply:
Check! Dan graag even een commecnt erboven waarom het uit gecomment is / hoe je het kan gebruiken.